features/netsocket/cellular/generic_modem_driver/TESTS/unit_tests/default/main.cpp

```diff
@@ -439,6 +439,8 @@ static void unlock()
 
 #else
 
+#error [NOT_SUPPORTED] MODEM_ON_BOARD should be set for this test to be functional
```

Comment:
I believe this can be even simplified, and print this error on the top of the file, this empty main is not required by our tools